### DISCUSSION TITLE
zone center unstable mode not getting stablilized

### DISCUSSION THREAD

**Discussion by matukumilli** [SSCHAcode/python-sscha#56] [Q&A] [answered]
Dear developers,
I am trying to calculate phonons for cubic SrTiO3 at 300K. Free energy got minimised after several populations of 2x2x2 supercell (after quite a large sample sizes towards the end). Now the free energy Hessian showing -ve freq corresponding to FE unstable mode at zone centre. AFD mode got stabilised well.
I checked the convergence of free energy Hessian calculation by splitting the final ensemble, and found no difference in frequencies. 
Why the FE unstable mode is not getting stabilised at 300K ? I am using PBEsol functional. How to make sure that effects of quantum fluctuations are incorporated correctly?

Regards,
Prasad

**Comment by ionerrea** [comment, marked as answer]
Dear Prasad,

Sometimes convergence of the frequencies coming from the free energy Hessian might be tricky and you may need more configurations. First of all make sure you are completely converged on the Hessian. 

Secondly, are you including the fourth-order terms in the Hessian calculation? If not, these terms, harder to calculate, may be responsible for the stabilization. You may need to check their role. Another question could be that with your particular functional the result is not good, or that your supercell (2x2x2) is not enough. 

Best regards,

**Comment by matukumilli** [comment]
Thank you, Dr. Ion Errea for your response. I will try including V4 term.

Regards,
Prasad
